gaj23/bigbirds
# Gaj/feature/manifest.json

#### Purpose: 
> Check all that apply.

### What does this PR do? 
> Clearly and concisely describe changes made. Include screenshots as needed.

This PR updates the manifest to reflect BigBirds brand. Additionally, custom icons were added rather than the boilerplate ones

<img width="398" alt="Screen Shot 2021-06-03 at 1 51 59 PM" src="https://user-images.githubusercontent.com/68332132/121247737-982e5e00-c870-11eb-9191-9eacebfd32f6.png">


### Problems Encountered & Resulting Solutions  
> State any problems encountered & include screenshots as needed. 
> Describe your code changes in detail for reviewers.

NA

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

See `public` for changed icons. 

### How should this be tested?
> What steps need to be taken to confirm functionality? 

When you open the browser website, you should see the new changes as displayed in the screen shot above.

### Steps To Take
> Any questions and possible steps moving forward.

Now, I would like to take the steps to make my app a PWA/

#### Checklist:
